lighthouse-core/scripts/run-git3po.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# requires a ghtoken set as GIT3PO_GH_TOKEN
+# likely invocation:
+#     env GIT3PO_GH_TOKEN=<token> lighthouse-core/scripts/run-git3po.sh
+
+set -ex
+
+DIRNAME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# git3po assumes relative config paths from cwd.
+cd "$DIRNAME"
+
+npm install -g git3po
+
+find git3po-rules/*.yaml -exec git3po -c {} \;
```

Comment:
love it

Reply:
we can now remove `startAt` from all the configs and do
```
--start-at=`date "+%s" -d "1 hour ago"`
```

although the autolock one should be special-cased to `1 month ago`

Reply:
k we do this in a followup.